data/relex-semantic.algs

```diff
@@ -1339,19 +1407,27 @@ TOBE_TO_LINKS
 ;
 ;  PREPOSITIONS
 ;
-;
-#TemplateActionAlg
-PREP_CLAUSE_LINKING_WHERE
-<LAB> = \WR\.*
-<F_L ref QUERY-TYPE> = %
-=
-<F_L head-word> += <F_R>
+;/////////////////////////////////////////////////////////////////////////////
 ;
 ; The rule below replaces the predeterminer with its object.
 ; Have to whack the existing object to do this.
 ; Example: "Joan thanked Susan for all the help she had given."
 ; We want to get for(thank, help) not for(thank, all)
 ;
+
+#TemplateActionAlg
+COMPLEX_PREPOSTION_CONSTRUCTION
+<LAB> = \Ju
+<F_R POS> = conjunction
+=
+<F_L ref links conj_prep> += <F_R ref>
+
+#TemplateActionAlg
+SO_THAT_STRUCTURE
+<LAB> = \EExk
+=
+<F_L str> = so_that

```

Comment:
This looks weird/wrong/bad to me: you are actually modifying the words in the sentence with this rule !??  Surely you don't want to modify the sentence itself!!